.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: php
+
+sudo: false
+
+env:
+  global:
+    - GITHUB_TOKEN=foo
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7
+    - php: hhvm
+ allow_failures:
+    - php: hhvm
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+before_install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then phpenv config-rm xdebug.ini; fi
```

Comment:
This helps speeding up the build and should conditionally be removed when interested in code coverage, of course.